examples/03_connectivity/plot_mni_spheres_extraction.py

```diff
@@ -0,0 +1,195 @@
+# -------------------------------------------
+#
+# We retrieve the first subject data of the ADHD dataset.
+from nilearn import datasets
+
+adhd_dataset = datasets.fetch_adhd(n_subjects=1)
+func_filename = adhd_dataset.func[0]
+confound_filename = adhd_dataset.confounds[0]
+
+# print basic information on the dataset
+print('First subject functional nifti image (4D) is at: {0}'.format(
+    func_filename))  # 4D data
+
+
+###############################################################################
+# We give coordinates of the posterior cingulate cortex,  the medial prefrontal
+# cortex and the left and right angular gyrus, all part of the DMN.
+pcc_coords = (1, -55, 17)
+mpfc_coords = (0, 51, -7)
+lag_coords = (-47, -71, 29)
+rag_coords = (50, -64, 27)
+
+dmn_coords = [pcc_coords, mpfc_coords, lag_coords, rag_coords]
+labels = ['PCC', 'MPFC', 'lAG', 'rAG']
+
+###############################################################################
+# It is advised to **mask the spheres with grey matter**. We don't have
+# subject's grey matter mask, we resort to a less precise group-level one.
+gm_mask_img = datasets.fetch_icbm152_brain_gm_mask()
+print(type(gm_mask_img))
+
+###############################################################################
+# This is a Nifti image, characterized by a data array, an affine and a header.
+
+###############################################################################
+# Creating the ROIs
+# -----------------
+#
+# Data in the gray mask image is stored in a 3D array.
+
+gm_mask_array = gm_mask_img.get_data()
+print('GM image data of shape {0}.'.format(gm_mask_array.shape))
+
+###############################################################################
+# We identify indices of GM voxels and compute their MNI coordinates.
+import numpy as np
+from nilearn.image.resampling import coord_transform
+i, j, k = np.where(gm_mask_array != 0)
+gm_voxels_coords = np.array(coord_transform(i, j, k, gm_mask_img.affine)).T
+
+print('Computed MNI coordinates of {0} GM voxels.'.format(
+    gm_voxels_coords.shape[0]))
+
+###############################################################################
+# We use the obtained coordinates to mask GM voxels more than 8mm from each
+# ROI center,
+gm_voxels_in_pcc = np.linalg.norm(gm_voxels_coords - pcc_coords, axis=1) < 8.
+gm_voxels_in_mpfc = np.linalg.norm(gm_voxels_coords - mpfc_coords, axis=1) < 8.
+gm_voxels_in_lag = np.linalg.norm(gm_voxels_coords - lag_coords, axis=1) < 8.
+gm_voxels_in_rag = np.linalg.norm(gm_voxels_coords - rag_coords, axis=1) < 8.
+
+print(gm_voxels_in_pcc)
+###############################################################################
+# and combine the masks to mark GM voxels part of the whole DMN.
+gm_voxels_in_dmn = \
+    gm_voxels_in_pcc + gm_voxels_in_mpfc + gm_voxels_in_lag + gm_voxels_in_rag
+
+print('{0} voxels from the {1} GM voxels are part of our DMN.'.format(
+    gm_voxels_in_dmn.sum(), gm_voxels_in_dmn.shape[0]))
```

Comment:
Do you mean it is not a good place for them here or in any part of nilearn examples ?
I thought it was important to visualize the spheric ROIs. All the other maskers take an ROIs file while the `NiftiSpheresMasker` creates its own ROIs that are in fact hidden.
For me visualizing my ROIs is important, for instance if there is an overlap I want to see it, also the difference between different radii, etc.
Also, I though it may be useful to demo the `coord_transform` function.